src/doc/rustdoc/src/unstable-features.md

```diff
@@ -84,6 +84,36 @@ in documentation.
 `#![feature(doc_cfg)]` feature gate. For more information, see [its chapter in the Unstable
 Book][unstable-doc-cfg] and [its tracking issue][issue-doc-cfg].
 
+Something important to be noted is that since [#89596](https://github.com/rust-lang/rust/pull/89596)
```

Comment:
Hmm, I'm not sure if we should be referencing PRs in the docs. It doesn't really add much.